test/nginx_system_test.sh

```diff
@@ -1263,6 +1277,10 @@ OUT=$(cat "$ERROR_LOG" \
     | grep -v "\\[error\\].*Failed to make directory*" \
     | grep -v "\\[error\\].*Could not create directories*" \
     | grep -v "\\[error\\].*opening temp file: No such file or directory.*" \
+    | grep -v "\\[error\\].*unexpected response.*" \
```

Comment:
re adding those lines is not needed. I can remove them.  I didn't see that
error on startup with the native fetcher on my tests before check in. I'll
take a look today.
On Mar 5, 2015 9:41 AM, "Otto van der Schaaf" <notifications@github.com>
wrote:

> In test/nginx_system_test.sh
> <https://github.com/pagespeed/ngx_pagespeed/pull/925#discussion_r25867491>
> :
>
> > @@ -1263,6 +1277,10 @@ OUT=$(cat "$ERROR_LOG" \
> >      | grep -v "\\[error\\].*Failed to make directory*" \
> >      | grep -v "\\[error\\].*Could not create directories*" \
> >      | grep -v "\\[error\\].*opening temp file: No such file or directory.*" \
> > +    | grep -v "\\[error\\].*unexpected response.*" \
>
> @crowell <https://github.com/crowell> was (re-)adding this line really
> needed? I was hoping that #922
> <https://github.com/pagespeed/ngx_pagespeed/pull/922> would have fixed
> this -- have you been testing with that change included?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pagespeed/ngx_pagespeed/pull/925/files#r25867491>.
>